### PR TITLE
Adds a safty lable to donk pockets

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Donk pockets after a legal battle have now placed a warning on the box
## Why It's Good For The Game

Was suggested as this still kills people from time to time + the food quirk making it much easyer to od on donks 

## Changelog
:cl:
add: New safety label has been added to Donk Pocket Box
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/boxes.dm [hunk 1]
```diff
@@ -431,7 +431,7 @@
 
 /obj/item/storage/box/donkpockets
 	name = "box of donk-pockets"
-	desc = "<B>Instructions:</B> <I>Heat in microwave. Product will cool if not eaten within seven minutes.</I>"
+	desc = "<B>Instructions:</B> <I>Heat in microwave. Product will cool if not eaten within seven minutes. WARING: Eating to many at once may lead to bodily harm!</I>"
```
Comment (ghost) [inline code/game/objects/items/storage/boxes.dm]: Spell Check: **too** many

Comment (ghost) [reply code/game/objects/items/storage/boxes.dm]: Also **Warning** instead of Waring

**Comment by ghost** [comment]
Spell check in title too <3

**safety label**

**Review by ghost — APPROVED** [review]

**Review by Ghommie — CHANGES_REQUESTED** [review]
It's very unlikely for someone to actually OD on omnizine by binging on warm donk pockets alone, the box only contains 6 units with 3u of ominize each, for a total of 18u vs the od threshold of 20, not including the click delay per bite and food intake cap...
But if that ever happens, that's a well deserved quality gameplay punishment for being too voracious folks, no need to try and idiotproof it.

**Comment by ghost** [comment]
Honestly... Ghommie does have a good point here. Maybe we should change the warning to something thats more towards scaliebait instead?

**Comment by Trilbyspaceclone** [comment]
It was suggested, personally I agree their should be a, just last week someone walked into sci, and was in crit do to donk OD. It does happen time to time

**Comment by ghost** [comment]
They have to eat more than a box of Donk to actually OD off the omnizine, that or they are a dumb scaliebait who ate them

**Comment by Ghommie** [comment]
EDIT: It's actually not so unlikely to OD on donk pockets with an upgraded microwave because upgraded machines are straight up capable of multiplying atoms and reagents like loaves and fishes. I'm fed up of it.

**Comment by ghost** [comment]
> EDIT: It's actually not so unlikely to OD on donk pockets with an upgraded microwave because upgraded machines are straight up capable of multiplying atoms and reagents like loaves and fishes. I'm fed up of it.

Nerf it upstream. Even if it's denied I would port it down here

**Comment by Trilbyspaceclone** [comment]
I thought they od'ed on 1 donk and  2 bites. Cuz the first and only time ive od on donks was with an upgraded microwave - note this like 3 years ago and now I tend to eat cooks food rather then vender trash if any at all

**Comment by ghost** [comment]
Wait I just realized this is Citadel tf am I doing here.

Why am I getting notifications for PRs on cit

**Comment by Ghommie** [comment]
Upgraded microwaves were nerfed, no need for a label anymore.

**Pull request closed without merging**